editoast/src/views/projects.rs

```diff
@@ -60,6 +60,11 @@ impl From<ProjectWithStudies> for ProjectWithImageUrl {
     }
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct QueryParams {
+    pub ordering: Option<OrderingProject>,
```

Comment:
We can use serde default instead of an option.

```suggestion
    #[sede(default = Ordering::LastModifiedAsc)]
    pub ordering: OrderingProject,
```